sparrow-py/pytests/collect_test.py

```diff
@@ -64,7 +64,7 @@ def test_collect_with_min(source, golden) -> None:
 def test_collect_with_min_and_max(source, golden) -> None:
     m = source["m"]
     n = source["n"]
-    golden(
+    golden.jsonl(
```

Comment:
May need to save the golden files again, looks like they were all deleted? Unless there are multiples